src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -257,6 +267,14 @@
     </{{ element|default('div') }}>
 {%- endblock form_row %}
 
+{% block file_row -%}
+    <{{ element|default('div') }} class="form-group custom-file">
```

Comment:
According to [docs](https://getbootstrap.com/docs/4.0/components/forms/#file-browser) and the provided example, form-group class is not needed.